src/USER-MISC/compute_pressure_cylinder.cpp

```diff
@@ -0,0 +1,493 @@
+/* ----------------------------------------------------------------------
+   LAMMPS - Large-scale Atomic/Molecular Massively Parallel Simulator
+   http://lammps.sandia.gov, Sandia National Laboratories
+   Steve Plimpton, sjplimp@sandia.gov
+
+   Copyright (2003) Sandia Corporation.  Under the terms of Contract
+   DE-AC04-94AL85000 with Sandia Corporation, the U.S. Government retains
+   certain rights in this software.  This software is distributed under
+   the GNU General Public License.
+
+   See the README file in the top-level LAMMPS directory.
+------------------------------------------------------------------------- */
+
+#include "math.h"
```

Comment:
please include "system headers", i.e those not provided by LAMMPS using angular brackets and also using C++ syntax (where supported by C++98). Example: `#include "math.h"` should become `#include <cmath>`